### ISSUE TITLE
Upcoming function deprecation in 32blit SDK

### ISSUE THREAD

**Issue by Gadgetoid** [swordlordcodingcrew/DANG#1]
Just as a heads up, I've merged the following PR:

https://github.com/32blit/32blit-sdk/pull/733

This deprecates:

```c++
blit(Surface *src, Rect r, Point p, bool hflip)
blit_sprite(const Rect &src, const Point &p, uint8_t t = 0)
stretch_blit_sprite(const Rect &src, const Rect &r, uint8_t t = 0)
```

This will, hopefully, be released as a new SDK version sometime this month.

It looks like you use `blit::screen.blit_sprite(src, dst, transform)` which should be replaced with `blit::screen.blit(blit::screen.sprites, src, dst, transform)` when you move to the updated SDK.

**Comment by LordEidi** [comment]
Ah, waited for that. Thanks for the heads up, as well as for the merge! Much appreciated. Let's remove a few ifs in our code then :)

**Comment by LordEidi** [comment]
Fixed with commit 34ca90c4cdbf642a612a5f49e1e049a31e2fc24e. Heads up to @LordFilu.
